3 reports.sql

```diff
@@ -0,0 +1,58 @@
+use flyairDB
+go
+
+-- 1) How many people are booked per flight as: count of people, flight number, departure airport, when the flight is departing and destination.
+
+select PassengersOnFlight= count(f.FlightNumber) , f.FlightNumber, f.DepartureAirport, f.DepartureCountry ,f.TimeDeparting, f.ArrivalAirport, f.ArrivalCountry
+from flyair f 
+group by f.FlightNumber, f.FlightNumber, f.DepartureAirport, f.DepartureCountry  , f.TimeDeparting, f.ArrivalAirport, f.ArrivalCountry
+
+
+--2) Who isn't checked in for flights departing in the next week, in order to send them reminders to check in.
+
+select *
+from flyair f 
+where f.CheckedIn = 0 
+and DATEDIFF(day, GETDATE(), f.DateDeparting)<=7
```

Comment:
Make sure to exclude flights that departed in the past.

Reply:
fixed it